src/components/pluginManager.js

```diff
@@ -1,37 +1,38 @@
-define(['events', 'globalize'], function (events, globalize) {
-    'use strict';
+import events from 'events';
+import globalize from 'globalize';
+/* eslint-disable indent */
 
     // TODO: replace with each plugin version
     var cacheParam = new Date().getTime();
 
-    function loadStrings(plugin) {
-        var strings = plugin.getTranslations ? plugin.getTranslations() : [];
-        return globalize.loadStrings({
-            name: plugin.id || plugin.packageName,
-            strings: strings
-        });
-    }
+    class PluginManager {
+        pluginsList = [];
 
-    function definePluginRoute(pluginManager, route, plugin) {
-        route.contentPath = pluginManager.mapPath(plugin, route.path);
-        route.path = pluginManager.mapRoute(plugin, route);
+        get plugins() {
+            return this.pluginsList;
+        }
 
-        Emby.App.defineRoute(route, plugin.id);
-    }
+        #loadStrings(plugin) {
+            var strings = plugin.getTranslations ? plugin.getTranslations() : [];
+            return globalize.loadStrings({
+                name: plugin.id || plugin.packageName,
+                strings: strings
+            });
+        }
 
-    function PluginManager() {
-        this.pluginsList = [];
-    }
+        #definePluginRoute(route, plugin) {
+            route.contentPath = this.#mapPath(plugin, route.path);
+            route.path = this.mapRoute(plugin, route);
```

Comment:
I can't get here
```suggestion
            route.path = this.#mapRoute(plugin, route);
```
or maybe just make it public?